conftest.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+import datetime
+from django.utils import timezone
+from message.models import Message, AuthorOptions
+from daycare.models import DayCare, Image
+from dogowner.models import DogOwner
+from orders.models import Order
+from review.models import Review
+
+
+@pytest.fixture
+def create_order():
+    return Order.create(dog_owner_id=DogOwner.objects.get(id=1),
+                        daycare_id=DayCare.objects.get(id=1),
+                        start_date=timezone.now(),
+                        end_date=timezone.now() + datetime.timedelta(days=3),
+                        price_per_day=100,
+                        )
+
+
+@pytest.fixture
+def dogowner_message_to_daycare1():
+    return Message.create(author=AuthorOptions.DogOwner,
+                          dogowner_id=DogOwner.objects.get(pk=1),
+                          daycare_id=DayCare.objects.get(pk=1),
+                          text='Hello this is the test message1 from owner to day care')
+
+
+@pytest.fixture
+def daycare1_reply_to_dogonwer_message():
```

Comment:
maybe `daycare1_reply_to_dogonwer1_message `

Reply:
same as above

Reply:
Please refrain from changing names in this PR when it is not mandatory. You can open a new PR for that once this one is merged.